### PR TITLE
added base MVVM structure and dependencies

### PR DESCRIPTION
**Pull request by MarcosCGlobant**
In this PR i added the base MVVM structure (packages, empty classes and layouts) for the project, along with some dependencies

packages

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".ui.meme_it_up.MemeItUpActivity">
```
Comment (SartiMau) [inline app/src/main/AndroidManifest.xml:12]: do not use **_** to separate words in packages, use **.**

File: app/src/main/java/com/example/memeitupapp/ui/listed_memes/ListedMemesFragment.kt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.example.memeitupapp.ui.listed_memes
```
Comment (SartiMau) [inline app/src/main/java/com/example/memeitupapp/ui/listed_memes/ListedMemesFragment.kt:1]: do not use **_** to separate words in packages, use **.**

File: app/src/main/java/com/example/memeitupapp/ui/meme_it_up/MemeItUpActivity.kt [hunk 3]
```diff
@@ -1,12 +1,13 @@
-package com.example.memeitupapp
+package com.example.memeitupapp.ui.meme_it_up
```
Comment (SartiMau) [inline app/src/main/java/com/example/memeitupapp/ui/meme_it_up/MemeItUpActivity.kt:1]: do not use **_** to separate words in packages, use **.**

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -4,7 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    tools:context=".ui.meme_it_up.MemeItUpActivity">
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml:7]: do not use **_** to separate words in packages, use **.**

File: app/src/main/res/layout/fragment_listed_memes.xml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.listed_memes.ListedMemesFragment">
```
Comment (SartiMau) [inline app/src/main/res/layout/fragment_listed_memes.xml:6]: do not use **_** to separate words in packages, use **.**

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (MarcosCGlobant) [reply app/src/main/AndroidManifest.xml:12]: done, thanks

File: app/src/main/java/com/example/memeitupapp/ui/listed_memes/ListedMemesFragment.kt (hunk 2 above)
Comment (MarcosCGlobant) [reply app/src/main/java/com/example/memeitupapp/ui/listed_memes/ListedMemesFragment.kt:1]: done, thanks

File: app/src/main/java/com/example/memeitupapp/ui/meme_it_up/MemeItUpActivity.kt (hunk 3 above)
Comment (MarcosCGlobant) [reply app/src/main/java/com/example/memeitupapp/ui/meme_it_up/MemeItUpActivity.kt:1]: done, thanks

File: app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (MarcosCGlobant) [reply app/src/main/res/layout/activity_main.xml:7]: done, thanks

File: app/src/main/res/layout/fragment_listed_memes.xml (hunk 5 above)
Comment (MarcosCGlobant) [reply app/src/main/res/layout/fragment_listed_memes.xml:6]: done, thanks

**Pull request merged**